changelogs/fragments/445-scaleway-improve_removal_handling.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+ - "scaleway_compute: Fix transition handling that could cause errors when removing a node (https://github.com/ansible-collections/community.general/pull/445)"
```

```suggestion
 - scaleway_compute - fix transition handling that could cause errors when removing a node (https://github.com/ansible-collections/community.general/pull/445).
```
https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to the latest changelog-how-to

---

Thanks for pointing this out. Made me realize that it also doesn't refer to the correct issue!